content/tutorials/generic/bootloader-reloading/bootloader-reloading.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Burn the bootloader on non-AVR boards
+description: Learn how to burn the bootloader on non-AVR boards.
+difficulty: Advanced
+featuredImage: 'clip'
+---
+
+In this tutorial, we will learn how to burn the bootloader on non-AVR boards using another Arduino board. The bootloader is a small piece of software that allows uploading sketches onto the Arduino board. It comes preprogrammed on the microcontrollers on Arduino boards. Whether the bootloader has been corrupted or intentionally has been removed, it can be restored by burning (also called, flashing or programming) a new bootloader to the board.
```

Comment:
```suggestion
title: Burn the bootloader on SAMD boards
description: Learn how to burn the bootloader on SAMD boards.
difficulty: Advanced
featuredImage: 'clip'
---

In this tutorial, we will learn how to burn the bootloader on boards that use the ATSAMD21G18 microcontroller using another Arduino board. The bootloader is a small piece of software that allows uploading sketches onto the Arduino board. It comes preprogrammed on the microcontrollers on Arduino boards. Whether the bootloader has been corrupted or intentionally has been removed, it can be restored by burning (also called, flashing or programming) a new bootloader to the board.
```

The instructions in this tutorial are specific to boards with the ATSAMD21 microcontroller. Since there are many non-AVR microcontrollers, the description of it being a tutorial for non-AVR boards was misleading